src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewProjectCmdlet.cs

```diff
@@ -71,13 +71,13 @@ public class NewProjectCmdlet : DataMigrationCmdlet
             Mandatory = true,
             HelpMessage = "Source platform type for project.")]
         [ValidateNotNullOrEmpty]
-        public ProjectSourcePlatform SourceType { get; set; }
+        public string SourceType { get; set; }
```

Comment:
Now that this is not an enum, do you want to add an argument completer here? It would allow the user to tab through a series of valid values.  This would take form of: PSCompleter("value1", "value2", "value3").

Reply:
Fixed.